hazelcast/src/main/java/com/hazelcast/internal/networking/nio/NioOutboundPipeline.java

```diff
@@ -52,6 +52,12 @@
         extends NioPipeline
         implements Supplier<OutboundFrame>, OutboundPipeline {
 
+    private enum State {
+        SCHEDULED,
+        UNSCHEDULED,
```

Comment:
Shouldn't we place the `UNSCHEDULE` as the first item in the enum? It would keep the `isScheduled`/`scheduled` probes more compatible.

Reply:
You are right.